core/templates/dev/head/components/question-directives/question-player/question-player.directive.html

```diff
@@ -226,10 +226,11 @@ <h3 class="oppia-question-player-header" translate="I18N_QUESTION_PLAYER_LEARN_M
   }
 
   .oppia-question-player-container .question-player-results-footer {
+    display: flex;
+    flex-wrap: wrap;
     height: 130px;
+    justify-content: space-around;
     margin-top: 16px;
-    padding: 0px 10px;
-    padding-top: 2px;
-    text-align: justify;
+    padding: 2px 10px 0;
```

Comment:
Why these changes?

Reply:
I am not sure why but it is the same in develop.

Reply:
Done

Reply:
It is corrected.